File: edge/script/code/temp.php
<?php
$inputFile = "/data/share/input_".getenv("xflow_node_id").".json";
$outputFile = "/data/share/output_".getenv("xflow_node_id").".json";
//从文件读取数据
$data = json_decode(file_get_contents($inputFile), true);
//自定义中间处理事件
$result = [];
foreach ($data as $item) {
    $result[]['url'] = $item['url'];
}

//将执行结果写入到文件中
file_put_contents($outputFile, json_encode($result, true));

File: edge/script/code/temp.py
import json
input_path = '/data/share/input.json'
output_path = '/data/share/output.json'
# 从文件中读取获得上游参数
with open(input_path, 'r') as f:
    data = json.load(f)

# 自定义中间事件
for key, value in data.items():
    # 这里处理你的逻辑
    print(value)

# 将结果输出到文件中
with open(output_path, 'w') as f:
    data = json.dump(data, f)